docs/content/cli/_index.md

```diff
@@ -1,4 +0,0 @@
----
```

Comment:
Let's undelete this file. My mistake! I now realize why the makefile target doesn't just delete that directory. This file helps us generate this file

https://porter.sh/cli/

which gives us a nice page with all of our commands.